packages/ui/vitest.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineProject, mergeConfig } from "vitest/config";
+import configShared from "../../vitest.shared.ts";
+
+export default mergeConfig(
```

Comment:
@nicholaslyon-dt added a config at the root and then merged it in the individual folders. I have some concern about the reliance on the shared config here. I might prefer going back to one config per workspace. Thoughts?

My understanding is that I can't automatically inherit from the root config. https://vitest.dev/guide/workspace#configuration

Reply:
Yeah I wasn't really thinking of the shared part as a 'config' per se, just a JS object that we could import for the reusable parts. But if this is the way they suggest in vitest, I'm fine with going back to one per workspace. It's easier to understand and sometimes that's best!